monas-content/src/infrastructure/crypto/hash/hmac_sha256.rs

```diff
@@ -0,0 +1,81 @@
+use hmac::{Hmac, Mac};
+use sha2::Sha256;
+
+#[derive(Debug, Clone, PartialEq, Eq)]
+pub enum HmacError {
+    KeyInitializationError,
```

Comment:
元になったエラーを保持するようにして、エラー原因を追跡できるようにすると良いです。

```suggestion
    KeyInitializationError(InvalidLength),
```